clang/test/Driver/sycl-device-lib-old-model.cpp

```diff
@@ -37,120 +37,6 @@
 // SYCL_DEVICE_LIB_LINK_DEFAULT-SAME: "{{.*}}libsycl-fallback-imf-fp64.bc"
 // SYCL_DEVICE_LIB_LINK_DEFAULT-SAME: "{{.*}}libsycl-fallback-imf-bf16.bc"
 
-/// ###########################################################################
```

Comment:
Why clear out the `-fsycl-device-lib-jit-link` testing?  We have only marked as deprecated with the intention to remove - but it hasn't been removed yet.

Reply:
Hi, @mdtoguchi 
I kept all tests untouched but found all test cases using this deprecated option would lead to build failure in pre-ci when some strict cflags are added such as "-Werror=xxx", so I cleared all these tests by removing the option.
Removing -fsycl-device-lib-ji-link testing in driver lit test is just for aligning for sycl e2e tests, do you think we should keep it for now and remove then until we finally removed this option? If so, I will revert the clang driver lit test change.

Thanks very much.

Reply:
@jinge90, yes please revert the driver test removal, as the functionality is still there until we do a full removal of the option and associated behavior.

Reply:
Hi, @mdtoguchi 
The driver lit test changes are reverted.
Thanks very much.